internal/deb/extract_test.go

```diff
@@ -482,7 +482,7 @@ func (s *S) TestExtractCreateCallback(c *C) {
 				relPath = relPath + "/"
 			}
 			sort.Slice(extractInfos, func(i, j int) bool {
-				return strings.Compare(extractInfos[i].Path, extractInfos[j].Path) < 0
+				return extractInfos[i].Path < extractInfos[j].Path
```

Comment:
[Comment for reviewer]: Per the go compile [source code](https://cs.opensource.google/go/go/+/refs/tags/go1.22.3:src/strings/compare.go;l=13):
```
	// NOTE(rsc): This function does NOT call the runtime cmpstring function,
	// because we do not want to provide any performance justification for
	// using strings.Compare. Basically no one should use strings.Compare.
```